client/.gitattributes

```diff
@@ -1,3 +1,3 @@
-pnpm-lock.yaml         binary
-vendor/                binary
+pnpm-lock.yaml         linguist-vendored linguist-generated
+**/vendor/**           linguist-vendored linguist-generated
```

Comment:
You can add `binary` in addition to vendored/generated, since that impacts diffs.